### ISSUE TITLE
治療終了者数のグラフをトップページに追加する

### ISSUE THREAD

**Issue by nanchovy** [codefortoda/covid19-saitama#58]
## 改善詳細 / Details of Improvement
- 治療終了者数の推移をグラフ化し、トップページに追加します

## スクリーンショット / Screenshot
完成予想図(右のグラフ)

![スクリーンショット 2020-04-12 19 41 12](https://user-images.githubusercontent.com/19761425/79066804-995c0d80-7cf5-11ea-8b01-66c31018f764.png)
[北海道 新型コロナウイルスまとめサイト](https://stopcovid19.hokkaido.dev/)より

## 期待する見せ方・挙動 / Expected behavior
- 陽性患者数は今後急激に伸びていくと予想されます。コロナウイルスから回復した人の数という可視化されづらいものをグラフ化することで、「どれくらいの人が感染しているのか」だけでなく、「どれくらいの人が治っているのか」を簡単に把握できるようになります。
- 現在患者数(上図左)をグラフ化すると「患者の数が減っているから外出しても大丈夫か」と考え、感染の第二波を誘発してしまう可能性があるかもしれません。そのため、グラフ化するのであれば、治療終了者数がより適切であると考えました。

同じようなissueが東京の対策サイトにも立っていました。
https://github.com/tokyo-metropolitan-gov/covid19/issues/2801

皆さんの意見をお伺いしたいです。

## 動作環境・ブラウザ / Environment
- macOS / Windows / Linux / iOS / Android
- Chrome / Safari / Firefox / Edge / Internet Explorer

**Comment by codefortoda** [comment]
埼玉県では最新の「退院（陰性確認済）」数を公開していますが、日別データは公開していませんので現時点では難しいように思います。
https://www.pref.saitama.lg.jp/a0701/shingatacoronavirus.html
※以前は、埼玉県のオープンデータ（発生状況）に陽性患者のステータス（退院等）が含まれておりましたが、プライバシー保護の観点から提供されないことになりました。
https://opendata.pref.saitama.lg.jp/data/dataset/covid19-jokyo

東京の issue 元となっている北海道では、オープンデータで「陰性確認済み数」を公開しています。
https://www.harp.lg.jp/opendata/dataset/1369.html

**Comment by nanchovy** [comment]
>埼玉県では最新の「退院（陰性確認済）」数を公開していますが、日別データは公開していませんので現時点では難しいように思います。

そうですね...。退院した人数のみの情報であればプライバシー保護の問題に抵触しないと思うので、一応埼玉県の方にも問い合わせしてみたいと思います。ただshowyouさんが #49 でおっしゃっているように、県の担当者の方が忙しいようで、オープンデータでの公開は難しそうですね。
